src/flb_config.c

```diff
@@ -56,6 +56,14 @@ struct flb_service_config service_configs[] = {
      FLB_CONF_TYPE_INT,
      offsetof(struct flb_config, grace)},
 
+    {FLB_CONF_STR_BACKOFF_BASE,
```

Comment:
config map first value is wrong, it aims to define the "value type", that value will lead to an undefined behavior.